### ISSUE TITLE
[STH] service is not used in database in mongo

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/IoT-STH#28]
Ex: 
Fiware-Service: my_service

In mongo:
```
sth_orion
```
sth prefix is correct, but `orion` is service by default

**Comment by gtorodelvalle** [comment]
Already fixed in #22 once it lands after you notice via Skype ;) Would you be so kind to close the issue if you agree, @iariasleon :) Thanks!

**Comment by iariasleon** [comment]
LGTM
```
> show dbs
local                   0.031GB
sth_loadtest_10_100_1   0.031GB
sth_loadtest_10_100_10  0.031GB
sth_loadtest_10_100_2   0.031GB
sth_loadtest_10_100_3   0.031GB
sth_loadtest_10_100_4   0.031GB
sth_loadtest_10_100_5   0.031GB
sth_loadtest_10_100_6   0.031GB
sth_loadtest_10_100_7   0.031GB
sth_loadtest_10_100_8   0.031GB
sth_loadtest_10_100_9   0.031GB
```
